src/audio-player.tsx

```diff
@@ -27,6 +32,12 @@ class AudioPlayer extends BasePlugin {
     this.prepareUI();
   }
 
+  loadMedia(): void {
+    Object.keys(this.player.plugins).forEach((plgName: string) => {
```

Comment:
if loadMedia is called again the same plugins will be added twice

Reply:
good point fixed (reset on reset())